src/createXcmCalls/reserveTransferAssets.ts

```diff
@@ -43,5 +46,21 @@ export const reserveTransferAssets = (
 		assetIds
 	);
 
-	return ext(dest, beneficiary, assets, 0);
+	let feeAssetItem = 0;
```

Comment:
Lets apply this logic to `createFeeAssetItem` for `ICreateXcmTypes` for each direction